# Request
int32 id_carrier
int32 id_station
float64 plc_timestamp

---
# Response
float64 process_time
